fonts/stylesheet.css

```diff
@@ -0,0 +1,48 @@
+/*! Generated by Font Squirrel (https://www.fontsquirrel.com) on September 12, 2020 */
+
+
+
+@font-face {
+    font-family: 'pt_serifbold';
+    src: url('ptserif-bold-webfont.woff2') format('woff2'),
+         url('ptserif-bold-webfont.woff') format('woff');
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6170621)